packages/api/helper/sequelize.js

```diff
@@ -0,0 +1,7 @@
+const { Sequelize } = require('sequelize');
+
+const sequelize = new Sequelize(
+  `postgres://${process.env.POSTGRES_USER}:${process.env.POSTGRES_PASSWORD}@${process.env.POSTGRES_PORT}/${process.env.POSTGRES_DBNAME}`
```

Comment:
In Heroku, these separate env vars won't exist. Instead, we'll have a single variable called `DATABASE_URL` that contains the whole connection string. What do you think about using the same pattern in our local setup?

Aside from that, what do you think about using a default value here for local development? Since everyone set up a `postgres` user during their project setup process, we could use that locally, which might help us avoid having to use a .env file for this part.